.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+updates:
+    - package-ecosystem: 'npm'
+      directory: '/'
+      schedule:
+          interval: 'weekly'
+          day: 'saturday'
+      versioning-strategy: 'increase'
+      labels:
+          - 'dependencies'
+      open-pull-requests-limit: 10
+      pull-request-branch-name:
+          separator: '-'
+      commit-message:
+          # cause a release for non-dev-deps
+          prefix: fix(deps)
+          # no release for dev-deps
+          prefix-development: chore(dev-deps)
+      ignore:
+          - dependency-name: '@salesforce/dev-scripts'
+          - dependency-name: '*'
+            update-types: ['version-update:semver-major']

```

Comment:
I'm reluctant to do it. In the past, every time `@oclif/core` or `@salesforce/core` had a major update it was breaking change and we needed to refactor our code to make it work.

Reply:
Those are actually my consideration. I tend to feel like it's better to know those breaking changes are coming, otherwise they tend to slip through the cracks for months until we realize we're at the end of the _minimum_ support, and then it's a whole pile of upgrade that we can't plan for as well.